packages/database/README.md

```diff
@@ -1,17 +1,21 @@
 This contains the database schema for vector embeddings and concepts.
-All CLI commands below should be run in this directory (`packages/database`.)
 
-1. Setup
+There are three usage scenarios:
+
+## Local development setup
+
+Normal scenario: Your backend and frontend will work against a database instance within docker.
+It does mean you will have a fresh database with minimal data.
+
+1. Installation
    1. Install [Docker](https://www.docker.com)
-   2. Install the [supabase CLI](https://supabase.com/docs/guides/local-development). (There is a brew version)
-   3. `supabase login` with your (account-specific) supabase access token. (TODO: Create a group access token.)
-   4. `supabase link`. It will ask you for a project name, use `discourse-graphs`. (Production for now.) It will also ask you for the database password (See 1password.)
-   5. Install [sqruff](https://github.com/quarylabs/sqruff)
+   2. Set the `SUPABASE_WORKDIR` in your environment to the absolute path of the `packages/database` directory.
```

Comment:
Where is `SUPABASE_WORKDIR` used? I couldn't find any references to it.

Reply:
It's used by the supabase executable, when running in the root directory.